docs/config-file-package-configuration-yml.md

```diff
@@ -1,38 +1,43 @@
-# The package configuration file (Experimental)
+# The Package Configuration File (Experimental)
 
-A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
-provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
-opposed to packages.
+A package configuration file allows you to define path excludes and license finding curations for a specific package (dependency) and 
+provenance. Conceptually, the file is similar to [.ort.yml](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-ort-yml.md), but it is used only for packages included via a package manager as project dependencies, and not for the project's own source code respository to be scanned.
 
-## Specifying the target package and provenance
+### When To Use
+Use a package configuration file to:
+
+-  mark files and directories as not included in released artifacts -- use it to make clear that license findings in documentation or tests in a package sources do not apply to the release (binary) artifact which is a dependency in your project
+-  overwrite scanner findings to correct identified license in a dependency for a specific file or directory
```

Comment:
overwrite -> override?

Reply:
identified license -> identified licenses

Reply:
remove 'or director' as this is not how it (currently) works.

Reply:
> overwrite -> override?

"Override" is the cancellation of some previous action or decision. "Overwrite" specifically refers to something being written over something previously written. In other words, actions and decisions are overriden. Information is overwritten. Furthermore it is overwritten with new information.

Source: https://english.stackexchange.com/questions/88400/when-to-use-override-and-overwrite